app/services/taxonomy/show_page.rb

```diff
@@ -70,15 +70,15 @@ def taxon_deletable?
 
     def email_subscribers
       @email_subscribers ||= begin
-        email_lists = {}
-
         begin
-          email_lists = Services.email_alert_api.find_subscriber_list(links: { taxon_tree: [taxon.content_id] })
+          email_lists = Services.email_alert_api.find_subscriber_list("links" => { taxon_tree: [taxon.content_id] })
+          email_lists.dig("subscriber_list", "active_subscriptions_count")
+        rescue GdsApi::HTTPNotFound
+          "0"
         rescue GdsApi::BaseError, SocketError => e
           GovukError.notify(e)
+          "?"
```

Comment:
what does it mean?

Reply:
It means it can't talk to the email-alert-api.  I opted for "?" rather than a sentence, because there's not much space:

<img width="1920" alt="screen shot 2018-05-31 at 16 37 52" src="https://user-images.githubusercontent.com/75235/40792121-061b79b6-64f1-11e8-97fb-8843a241074f.png">